dbft.go

```diff
@@ -201,6 +201,10 @@ func (d *DBFT) OnTimeout(hv timer.HV) {
 
 // OnReceive advances state machine in accordance with msg.
 func (d *DBFT) OnReceive(msg payload.ConsensusPayload) {
+	if d.BlockSent() {
```

Comment:
Suppose you're the only node with this block. Now you get a recovery request from someone else.